orsopy/fileio/base.py

```diff
@@ -91,6 +91,36 @@ def _custom_init_fn(fieldsarg, frozen, has_post_init, self_name, globals):
 
 
 def orsodataclass(cls: type):
+    if os.environ.get("GENERATE_SCHEMA", False) == "True":
```

Comment:
I don't have experience working with Pydantic and how to work with it, but it seems to me that creating a schema is similar to running unit test. So I would prefer having code that does these sort of changes separated from the core library as it reduces coupling.
Do you think this setup can be done in a script before importing the first orsopy.fileio modules?

Reply:
See comment on header_schema.py